numpy/core/src/multiarray/datetime_busday.c

```diff
@@ -396,7 +401,7 @@ apply_business_day_count(npy_datetime date_begin, npy_datetime date_end,
                                         holidays_begin, holidays_end);
 
     /* Start the count as negative the number of holidays in the range */
-    count = -(holidays_end - holidays_begin);
+    count += -(holidays_end - holidays_begin);
```

Comment:
I think this change and the change to initialize count to 0 are not necessary. Just incrementing the dates is enough.